src/products/android-l10n/README.md

```diff
@@ -0,0 +1,4 @@
+# L10n and android-l10n
```

Comment:
Does this need to be hooked up in `SUMMARY.md` ?

Reply:
Good catch. Yes it should